CHANGELOG.md

```diff
@@ -1,27 +1,222 @@
+
+Result-set
+
+- Degrading performance when modifying multiple rows concurrently in result-set [#2007](https://github.com/oceanbase/odc/pull/2007)
+- In OceanBase MySQL mode, the datetime data type loses precision when displayed [#1996](https://github.com/oceanbase/odc/pull/1996)
+- Switching between viewing BLOB field text and hexadecimal images may cause the UI to freeze [#300](https://github.com/oceanbase/odc/issues/300)
 
+Table Object
 
+- Disparity in column names order between Index and Constraint views [#1948](https://github.com/oceanbase/odc/pull/1948)
+- Inability to view table details for MySQL version 5.6 [#1635](https://github.com/oceanbase/odc/pull/1635)
+- Sofa ODP table detail view not accessible [#2043](https://github.com/oceanbase/odc/pull/2043)
+- Unable to change NOT NULL fields to NULL in table structure editor [#1441](https://github.com/oceanbase/odc/issues/1441)
+- When partitioned table has multiple maximum values, only one is displayed [#1501](https://github.com/oceanbase/odc/issues/1501)
+- Button to delete table primary key is greyed out and not clickable [#1874](https://github.com/oceanbase/odc/issues/1874)
 
+Import/Export
+
+- SQL statement split incorrectly causing task failure when comments contain `;` [#417](https://github.com/oceanbase/odc/issues/417)
+- Export task fails when type name is in lowercase [#631](https://github.com/oceanbase/odc/issues/631)
+- Exporting trigger objects fails [#750](https://github.com/oceanbase/odc/issues/750)
+- Export task fails when function name contains special characters [#1331](https://github.com/oceanbase/odc/issues/1331)
+- During Oracle mode export of indexes, index names are prefixed with database name [#1491](https://github.com/oceanbase/odc/issues/1491)
+- `DELIMITER $$` separator sticks to table name when exporting procedure structure [#1746](https://github.com/oceanbase/odc/issues/1746)
+- After creating an export task, terminating it shows task status as executed successfully [#1752](https://github.com/oceanbase/odc/issues/1752)
+- When exporting packages, object type in task details doesn't show the package body [#1755](https://github.com/oceanbase/odc/issues/1755)
+- Import task fails when CSV file includes DATE type [#2079](https://github.com/oceanbase/odc/issues/2079)
+
+Online Schema Change
+
+- OSC task experiencing syntax exceptions when input statements contain comments [#1597](https://github.com/oceanbase/odc/pull/1597)
+
+Project and ticket
+
+- Ticket creation success message incorrect, changed to 'Ticket create successfully' [#1320](https://github.com/oceanbase/odc/issues/1320)
+- SQL window dropdown switch project causing page crashes [#1512](https://github.com/oceanbase/odc/issues/1512)
+
+Database Changes
+
+- If rollback content is attachments, the rollback content are not displayed during the rollback process [#1379](https://github.com/oceanbase/odc/issues/1379)
```

Comment:
### Projects and Tickets

- Incorrect prompt message for successful task creation, changed to "Ticket Created Successfully". [#1320](https://github.com/oceanbase/odc/issues/1320)
- Dropdown switching project page crashes under SQL window. [#1512](https://github.com/oceanbase/odc/issues/1512)

Database Changes

- When the rollback content is an attachment, it is not displayed during the rollback process. [#1379](https://github.com/oceanbase/odc/issues/1379)